python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -406,6 +406,14 @@ def resolve_repository(self, graphene_info) -> "GrapheneRepository":
         )
 
 
+class GrapheneAssetGroup(graphene.ObjectType):
+    groupName = graphene.String()
```

Comment:
Hey @shalabhc! I think this should be a `graphene.NonNull(graphene.String)`? Client-side this autogenerates `groupName: string | null` and I don't think it should ever be null?